avm/res/network/network-interface/tests/e2e/waf-aligned/main.test.bicep

```diff
@@ -0,0 +1,121 @@
+targetScope = 'subscription'
+
+metadata name = 'Using large parameter set'
+metadata description = 'This instance deploys the module with most of its features enabled.'
```

Comment:
Plaese update to the WAF naming & description & refresh the readme

Reply:
Done